src/index.js

```diff
@@ -152,8 +153,7 @@ const init = async () => {
 
     server.listen(config.hostingEnvironment.port, () => {
       logger.info(
-        `Dev server listening on https://${config.hostingEnvironment.host}:${
-          config.hostingEnvironment.port
+        `Dev server listening on https://${config.hostingEnvironment.host}:${config.hostingEnvironment.port
```

Comment:
do we still have hosting env in the config now after the changes?😺

Reply:
yes, only moved asset bit out of it